src/kakarot/instructions/system_operations.cairo

```diff
@@ -76,11 +76,9 @@ namespace SystemOperations {
         Memory.load_n(size.low, bytecode, offset.low);
 
         // Get target address
-        let evm_contract_address = CreateHelper.get_evm_address(
-            evm.message.address, popped_len, popped, size.low, bytecode
+        let target_evm_address = CreateHelper.get_evm_address(
```

Comment:
this is called `contract_address` in EELS so I think that we should either leave it as it was or just remove the `evm_` (see also my other comment regarding the fact that addresses should just be EVM by default, https://github.com/ethereum/execution-specs/blob/559ba54829a5d726ca95fe5701486f7bd303bf7f/src/ethereum/shanghai/vm/instructions/system.py#L163C5-L163C21)